linkedList/homework2.php

```diff
@@ -5,15 +5,19 @@ class LinkedList3 extends LinkedList
 {
     //made function without following hints at first
     public function reverse() {
+        if ($this->head == null) {
+            echo "List is empty";
+        }
+
         $currNode = $this->head; 
         $arr = [];  
         while ($currNode != null) {
             array_push($arr, $currNode->getData());
             $currNode = $currNode->getNext();
         }
         $arr2 = [];
-        for ($i=0; $i<sizeof($arr); $i++) {
-            $t =100; 
+        for ($i=sizeof($arr2)-1; $i>=0; $i--) {
+            $t =$arr[$i]; 
             $t = $arr[(sizeof($arr)-1-$i)];
```

Comment:
Actually, you must remove this line.
So, Could you please test again after changing codes?

Reply:
I remove the sentence you pointed out.
and I made some small modifications of function `reverse`.